gcip/core/variables.py

```diff
@@ -23,7 +19,7 @@ class PredefinedVariables():
     https://docs.gitlab.com/ee/ci/variables/predefined_variables.html
     """
 
-    CHAT_CHANNEL: str = EnvProxy("CHAT_CHANNEL")
+    CHAT_CHANNEL: EnvProxy = EnvProxy("CHAT_CHANNEL")
```

Comment:
This annotation is not useful, if the user wants to know what `CHAT_CHANNEL`s datatype is, he gets EnvProxy. 
Mybe whe can stop checking this class or we can e.g. cast EnvProxy to str with the `typing` module?

Reply:
For now, this is ok. 
In a future release we will change this to be a classproperty.
See: https://stackoverflow.com/questions/128573/using-property-on-classmethods´